TEMPLATES/.hadolint.yaml

```diff
@@ -3,3 +3,9 @@
 ## Hadolint config file ##
 ##########################
 ignored:
+    # Rule to pin versions in apt-get install
```

Comment:
Thanks for putting these comments here! They're super helpful without having to refer to the documentation every time to figure out what they do.